src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1920,4 +1926,56 @@ public Response getBannerMessages(@PathParam("id") Long id) throws WrappedRespon
 
     }
     
+    @GET
+    @Path("/licenses")
+    public Response getLicenses() {
+        return ok(licenseService.listAll().stream()
+        		.map(JsonPrinter::json)
+        		.collect(toJsonArray()));
+    }
+
+    @GET
+    @Path("/licenses/{id}")
+    public Response getLicense(@PathParam("id") long id) {
+    	License l = licenseService.get(id);
```

Comment:
Don't use the lowercase l as a variable name, it looks too much like the digit 1 in a lot of fonts. I suggest lic if you want to keep it short.

Reply:
Also here it's not my own preference, I personally prefer using `license`. I just kept consistent with the class `SettingsServiceBean` where you'll see code like `Setting s`. I'll change it to `license` since there's other service beans with these longer names.